### ISSUE TITLE
Implement Go_to_Top button and Go_To_Bottom button on the feed page

### ISSUE THREAD

**Issue by greedynamic** [eddyspaghette/TreasureFind#69]

**Comment by akn0717** [comment]
I think it is better just to have go to top, go to bottom is kind of heavy if there are like 1000 sessions on the database

**Comment by greedynamic** [comment]
> I think it is better just to have go to top, go to bottom is kind of heavy if there are like 1000 sessions on the database

Agree. But i don't want to implement it now since it's not a major feature. I'll code it maybe after due.
